infrastructure/template.yaml

```diff
@@ -1417,6 +1418,7 @@ Resources:
           AUDIT_EVENT_BUS: !Ref CheckHmrcEventBus
           AUDIT_SOURCE: !FindInMap [EnvironmentConfiguration, !Ref Environment, DOMAINNAME]
           AUDIT_ISSUER: !Sub "{{resolve:ssm:/${CommonStackName}/verifiable-credential/issuer}}"
+          LOG_FULL_ERRORS: !If [IsProdEnvironment, "false", "true"]
```

Comment:
This is interesting because it feels like we should be able to achieve this with log levels. I'm not sure we're actually at a stage right now where we could safely rely on log levels (I'd probably probably want to do an audit of what we have so first) happy with this for now.

That being said, please could we update the condition to also included integration? It is a prod like environment, we have RPs testing in there and in theory there could be sensitive user data we need to protect